tests/tests_facts.yml

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: MIT
+---
+- hosts: all
+  name: Ensure that basic features of role work correctly
+  roles:
+    - linux-system-roles.systemd
+  tasks:
+    - name: Get units facts
```

Comment:
You run the role above, and it should collect unit facts by itself, so no need to run `systemd_units`, nor shall we support running it separately from the role.